solo-chains/runtime/starlight/src/lib.rs

```diff
@@ -1703,6 +1707,140 @@ impl pallet_state_trie_migration::Config for Runtime {
     type MaxKeyLen = MigrationMaxKeyLen;
 }
 
+pub struct NoRelayStorageRoots;
```

Comment:
I would implement this in tp_traits itself for (), what do you think?

Reply:
I would rather not, because that implementation will be wrong most of the time, it doesn't seem a good default value